### PR TITLE
Fix #5

### LINKED ISSUE #5: set the scales by setting the fourier periods
Allow this behaviour:

```
wt.fourier_periods = array_of_periods
```

The scales of a transform can currently be set by

```
wt = WaveletTransform(data)
wt.scales = array_of_scales
```

The equivalent fourier period is available as a property `wt.fourier_periods`,
however we can't set the scales via the fourier_periods. This would be useful
as the equivalent fourier period allows us to compare wavelets.

### PR DESCRIPTION
**Pull request by ulikoehler**
This PR fixes #5 and adds getters and setters for frequencies.

### CODE REVIEW

**Comment by aaren** [comment]
Thanks for this. Will take a look when I've time.

**Comment by lovechang1986** [comment]
This is the Crosswavelet  code and examples in matlab, would you interesting to coding in Python?
url:http://noc.ac.uk/using-science/crosswavelet-wavelet-coherence

File: wavelets/transform.py [hunk 1]
```diff
@@ -253,10 +253,38 @@ def fourier_period(self):
...
         """Return the equivalent fourier periods for the scales used."""
         return self.fourier_period(self.scales)
 
+    @fourier_periods.setter
+    def fourier_periods(self, periods):
+        """Set the scales based on a list of fourier periods"""
+        self.scales = self.wavelet.scale_from_period(periods)
+
+    @property
+    def frequencies(self):
```
Comment (aaren) [inline wavelets/transform.py]: I would prefer `fourier_frequencies` to avoid confusion with the angular frequencies `w_k`.

**Comment by ulikoehler** [comment]
@aaren fourier_frequencies is fine with me, I'll change that ASAP.

**Comment by aaren** [comment]
Two more things:

1. Can you add tests that exercise the new functionality?

2. Can you rebase onto current master? I merged some changes including automated testing, so we can try that out on this PR.

**Comment by aaren** [comment]
Were the print calls all it took to work with python 3? I'll add that to the test matrix.

**Comment by ulikoehler** [comment]
@aaren I have not tested everything yet, but it works flawlessly with Python 3.4. I'll rebase and add the tests.

**Comment by ulikoehler** [comment]
OK, should be done now

**Comment by aaren** [comment]
close / open to trigger travis build (now with python 3 tests)

**Comment by aaren** [comment]
Looks good to me now. Thanks for helping out @ulikoehler :)

**Comment by ulikoehler** [comment]
Sure thing ;-) Of course I'll try to continue working on this project when I have some spare time!

**Pull request merged**